stock_indicators/indicators/chandelier.py

```diff
@@ -16,7 +17,7 @@ class ChandelierResult(ResultBase):
     """
 
     @property
-    def chandelier_exit(self):
+    def chandelier_exit(self) -> Optional[Decimal]:
```

Comment:
Not sure if it's here or in one of the other Chandelier data types.  You'll have to add the latest v1.20.2 to see which is broken in your local version.  The production build uses the latest DLL.